Servers/UI/OJS.Servers.Ui/Controllers/AccountController.cs

```diff
@@ -85,9 +78,6 @@ public async Task<IActionResult> Login([FromBody]LoginRequestModel model)
                 return this.Unauthorized(GlobalConstants.ErrorMessages.InvalidUsernameOrPassword);
             }
 
-            var roles = await this.userManager.GetRolesAsync(user);
-
-            this.HttpContext.AppendAuthInfoCookies(roles, user.UserName);
```

Comment:
We should delete the implementation of `AppendAuthInfoCookies` as well as constants and other cookie related code.